src/app/directives/drag-and-drop.directive.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  Directive,
+  HostBinding,
+  HostListener,
+  Output,
+  EventEmitter
+} from "@angular/core";
+import { DomSanitizer, SafeUrl } from '@angular/platform-browser';
+
+export interface FileHandle {
```

Comment:
Please move this interface to the appropriate folder

Reply:
fixed